x-pack/plugins/actions/server/__jest__/create_fire_function.test.ts

```diff
@@ -4,158 +4,68 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 
-import { ActionTypeRegistry } from '../action_type_registry';
+import { taskManagerMock } from '../../../task_manager/task_manager.mock';
 import { createFireFunction } from '../create_fire_function';
 
-const services = {
-  log: jest.fn(),
-};
-const actionTypeRegistryParams = {
-  services,
-};
+const mockTaskManager = taskManagerMock.create();
 
-const mockEncryptedSavedObjects = {
-  isEncryptionError: jest.fn(),
-  registerType: jest.fn(),
-  getDecryptedAsInternalUser: jest.fn(),
+const savedObjectsClient = {
```

Comment:
Could you use the `SavedObjectsClient` mock src/core/server/saved_objects/service/saved_objects_client.mock.ts here?

Reply:
Good catch, fixed in https://github.com/elastic/kibana/pull/37043/commits/f3d4dc4fe6ba85630ac50456838d656c1097506e.